### DISCUSSION TITLE
thanhle.blog/day-la-mot-bai-viet-so-sai-nhung-doc-thu-coi

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#55] [Ideas]
# thanhle.blog/day-la-mot-bai-viet-so-sai-nhung-doc-thu-coi

Suy nghĩ Agile khi làm product hay bất cứ thứ gì đó

https://thanhle.blog/blog/day-la-mot-bai-viet-so-sai-nhung-doc-thu-coi

**Comment by nguyenhoaikhang37** [comment]
Nào trả nợ xong làm lại bài viết về kĩ thuật anh ơi, lâu quá mới thấy anh ra blog

**Reply by thanhlmm** [reply]
mua iPad để giúp ra nhiều bài viết kĩ thuật hơn :))))

**Reply by haulinh** [reply]
Bán ipad cũ để mua ipad mới hả anh :))))

**Reply by thanhlmm** [reply]
uớc j dc z, nhưng a bán trả nợ 😭
